google/cloud/bigquery/enums.py

```diff
@@ -28,6 +28,46 @@ class AutoRowIDs(enum.Enum):
     GENERATE_UUID = enum.auto()
 
 
+class BiEngineReasonCode(enum.Enum):
```

Comment:
I see very little reason why we'd want enums for this. It introduces a problem if the backend adds new codes we can have client-side code start failing.

At most, I'd suggest a list of string constants (basically a pretend enum)

Reply:
I removed the enums and just made them string constants.